crates/marine-test-macro-impl/src/marine_test/token_stream_generator.rs

```diff
@@ -17,14 +17,16 @@
 mod methods_generator;
 mod methods_generator_utils;
 mod record_type_generator;
+pub mod service_generator;
```

Comment:
Why is it pub?

Reply:
Because it contains `generate_app_service_ctor` and `generate_service_definitions` which are used in `glue_code_generator.rs`. I can `pub use` these functions or reorganize code, so they are defined in `token_stream_generator.rs`. Which one suits you better?

Reply:
I see, I think both are ok, so it's up to you.